spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -1848,6 +1849,21 @@ public Boolean isMandatoryFor(final Message message) {
 
 	@Override
 	public <T> T invoke(OperationsCallback<T> action) {
+		return invoke(action, null, null);
+	}
+
+	/**
+	 * Invoke operations on the same channel.
+	 * If callbacks are needed, both callbacks must be supplied.
+	 * @param action the callback.
+	 * @param acks a confirm callback for acks.
+	 * @param nacks a confirm callback for nacks.
+	 * @param <T> the return type.
+	 * @return the result of the action method.
```

Comment:
`@since 2.1` ?

Reply:
We need to add this method to `RabbitOperations` - can you do that if you find no more problems in the review?

Reply:
Oops - too late.

Reply:
I think we can just add subsequent commit.
Not a big deal though.

Thanks for the pointer! My bad :smile:

Reply:
Addressed via https://github.com/spring-projects/spring-amqp/commit/bf5498db8025be7c604b6a55804145de7e5a47d2